.pre-commit-config.yaml

```diff
@@ -24,8 +24,10 @@ repos:
       - id: codecov-config
         name: codecov-config-check
         description: Check if the codecov config file is valid
-        files: .github/.codecov.yml
-        entry: |
-          sh -c "curl --data-binary @.github/.codecov.yml \\
+        files: ".github/.codecov.yml"
+        entry: >
+          sh -c "curl -V --keepalive-time 60 \\
+                 --data-binary @.github/.codecov.yml \\
```

Comment:
This appears to be a known issue that local pre-commit hooks don't run correctly on Github Actions. I'm still inclined to keep this in the hook, on the off chance someone wants to modify the `.codecov.yml` file. OTOH, I'd don't like having red X marks indicating pre-commit.ci failure due to this. Perhaps, the solution is to not using `pre-commit.ci` bot?

Reply:
My only experience with pre-commit is local usage, and I'm a little wary of having anything actually modify the code itself in CI, if that's what it's supposed to do here.

Reply:
Fair enough. `pre-commit ci` is expected to add a new commit with formatting only changes (based on the config file setup). I'll revert to keeping this config file but remove the `pre-commit ci` bot from the repo.